Library/Formula/syncthing.rb

```diff
@@ -2,6 +2,8 @@ class Syncthing < Formula
   homepage "http://syncthing.net"
   url "https://github.com/syncthing/syncthing.git", :tag => "v0.10.24"
```

Comment:
Can you add the ` :revision ` (commit) to this ` :tag `? Looks like it was ` :revision => "e22c873ec4a65cabf1df5b09c343cbf4ea2325aa" `. Thanks!